src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,35 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        //Excavator excavator = new Excavator();
+        //Bulldozer bulldozer = new Bulldozer();
+        //Truck truck = new Truck();
+
+        Object[] object = new Object[3];
+
+        object[0] = new Excavator();
+        object[1] = new Bulldozer();
+        object[2] = new Truck();
+
+
+        for (Object objects : object ) {
+            //Excavator
+            Excavator.doWork();
+            Excavator.stopWork();
+            //Bulldozer
+            Bulldozer.doWork();
+            Bulldozer.stopWork();
+            //Truck
+            Truck.doWork();
+            Truck.stopWork();
```

Comment:
The loop should iterate over the instances of 'Machine' and call 'doWork' and 'stopWork' on each of them. Replace the explicit calls to 'Excavator', 'Bulldozer', and 'Truck' with a loop that does something like: 'for (Machine machine : machines) { machine.doWork(); machine.stopWork(); }'.